app/views/shared/provider/_flash.html.erb

```diff
@@ -1,13 +1,14 @@
-<div id="flashWrapper" style="<%= flash.empty? ? "display:none;" : "" %>">
+<div id="flashWrapper" style="position: fixed; margin-top: 2rem">
```

Comment:
but should still be display none when flash is empty, no?

Reply:
@thomasmaas is a WIP, I asked @HemaHG to make a PR to be able to test

Reply:
I agree with @thomasmaas 

```suggestion
<div id="flashWrapper" style="<%= flash.empty? ? "display:none;" : "position: fixed; margin-top: 2rem" %>">
```

Reply:
@HemaHG instead of adding the style inline, you can add it in `app/assets/stylesheets/provider/_flash.scss`
https://github.com/3scale/porta/blob/master/app/assets/stylesheets/provider/_flash.scss

keep the ruby code: `<%= flash.empty? ? "display:none;" : "" %>` 
And put the css in **_flash.scss**